tests/settings.py

```diff
@@ -161,3 +161,8 @@
     from local_settings import *
 except ImportError:
     pass
+
+REST_AUTH_SERIALIZERS = {
+    'PASSWORD_RESET_SERIALIZER':
+        'openwisp_radius.api.serializers.PasswordResetSerializer',
```

Comment:
I'd write this on one line for consistency with the rest of the settings, we allow `110` characters line length if I'm not wrong